src/utility/LG_internal.h

```diff
@@ -272,6 +272,17 @@ typedef unsigned char LG_void ;
 
 #endif
 
+// GraphBLAS version 10 flag
+#if LAGRAPH_SUITESPARSE 
+    #if GxB_IMPLEMENTATION >= GxB_VERSION (10,0,0)
+        #define USING_GRAPHBLAS_V10 1
+    #else
+        #define USING_GRAPHBLAS_V10 0
```

Comment:
Since this #define is internal, but accessible to any LAGraph algorithm, let's rename it to "LG_SUITESPARSE_GRAPHBLAS_V10".  That way, it mimics the LAGRAPH_SUITESPARSE definition (which is in LAGraph.h), but it has an LG_* prefix which means it's for internal use only.